RJL.UIP.HW3.ConditionalsMethods/Program.cs

```diff
@@ -0,0 +1,261 @@
+            if (minIndexInRandomArray > maxIndexInRandomArray)
+            {
+                tempIndex = minIndexInRandomArray;
+                minIndexInRandomArray = maxIndexInRandomArray;
+                maxIndexInRandomArray = tempIndex;
+            }
+            for (int i = minIndexInRandomArray + 1; i < maxIndexInRandomArray; i++)
+            {
+                randomArrayTask6[i] = 0;
+            }
+            Console.WriteLine("Array with null value between min and max is");
+            PrintArray(randomArrayTask6);
+            Console.WriteLine();
+            Console.WriteLine("-------------------------------------------------------------");
+            //task7
+            Console.WriteLine("Task7");
+            int randomArrayLengthTask7 = randomizer.Next(1, 10);
+            int[][] randomArrayTask7 = new int[randomArrayLengthTask7][];
+            Console.WriteLine("Current random array of arrays is");
+            for (int i = 0; i < randomArrayLengthTask7; i++)
+            {
+                randomArrayTask7[i] = CreateRandomArray();
+                PrintArray(randomArrayTask7[i]);
+                Console.WriteLine();
+            }
+            Console.WriteLine("Array after sorting");
+            for (int i = 0; i < randomArrayLengthTask7; i++)
+            {
+                QuickSortArray(randomArrayTask7[i], 0, randomArrayTask7[i].Length - 1);
+                PrintArray(randomArrayTask7[i]);
+                Console.WriteLine();
+            }
+            Console.WriteLine("-------------------------------------------------------------");
+            //task8
+            Console.WriteLine("Task8");
+            string[] multiplyTableArrayTask8 = GetMultiplyTable(15);
+            foreach (string item in multiplyTableArrayTask8)
+            {
+                Console.WriteLine(item);
+            }
+            Console.ReadLine();
+        }
+
+        static double GetFactorial(int inputIntFactorial)
+        {
+            double resultFactorial = (inputIntFactorial == 1)?1:(inputIntFactorial * GetFactorial(inputIntFactorial - 1));
+            return resultFactorial;
+        }
+        static int[] GetFibonachiSequence(int inputIntFibonachi)
+        {
+            int[] resultFibonachi = new int[inputIntFibonachi];
+            for (int i = 0; i <inputIntFibonachi; i++)
+            {
+                resultFibonachi[i] = (i <= 1) ? i : (GetFibonachiSequence(inputIntFibonachi-2)[i-2] + GetFibonachiSequence(inputIntFibonachi-1)[i-1]);
+            }
+            return resultFibonachi;
+        }
+        static int GetCountDaysInMonthYear(int year, int month)
+        {
+            int resultcountdays = 0;
+            if (month != 2&&month<13)
+            {
+                resultcountdays = (month % 2 != 0 && month < 8) || (month % 2 == 0 && month >= 8) ? 31 : 30;
+            }
+            if (month == 2)
```

Comment:
in your case only one if can be resolved - it means you should use **if else** or **if else if** etc